src/email-body.js

```diff
@@ -0,0 +1,66 @@
+const bodyStyle = `background: #60ADD6;
```

Comment:
Hi @des-des - How should I go about this? Please could you give me some more guidance. Is it just the CSS or should I be creating this HTML string in the front-end with handlebars?